third_party_update.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
```

Comment:
This seems to only update the flamegraph directory, so maybe name the file as flamegraph_update.sh or something like that?